pallets/investments/src/tests.rs

```diff
@@ -2781,7 +2783,7 @@ fn collecting_fully_works() {
 				INVESTMENT_0_0
 			));
 			assert_eq!(
-				n_last_event(3),
+				n_last_event(4),
```

Comment:
The expected event is behind another produced event of killing an account.

I think the rounding issues put the account under the existential deposit value, and the account is killed.